### PR TITLE
add vhost to route params

### PR DESCRIPTION
**Pull request by yuval-k**
BOT NOTES: 
resolves https://github.com/solo-io/gloo/pull/766

### CODE REVIEW

File: projects/gloo/pkg/translator/translator_test.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	"github.com/gogo/protobuf/proto"
-
+	envoyroute "github.com/envoyproxy/go-control-plane/envoy/api/v2/route"
```
Comment (marcogschmidt) [inline projects/gloo/pkg/translator/translator_test.go]: Nit: this is a duplicate of the import below.

**Comment by marcogschmidt** [comment]
Looks good! Just a small comment.

File: projects/gloo/pkg/translator/translator_test.go (hunk 1 above)
Comment (yuval-k) [reply projects/gloo/pkg/translator/translator_test.go]: goot catch

**Comment by yuval-k** [comment]
@marcogschmidt fixed!

**Pull request merged**